components/media/VideoPlayer.stories.js

```diff
@@ -0,0 +1,24 @@
+import { storiesOf } from '@storybook/vue';
+
+import VideoPlayer from './VideoPlayer.vue';
+
+storiesOf('Media', module)
+  .add('Video Player', () => ({
```

Comment:
Let's make the "story" more indicative of a specific use of the component:
```js
.add('WebM video', () => ({
```
... and also add a second story with an H.264 MP4, using the URL: https://europeana1914-1918.s3.amazonaws.com/attachments/236696/9555.236696.original.mp4